server.js

```diff
@@ -177,7 +176,7 @@ vendorDomain.on('error', function(err) {
 });
 
 
-function cache(f) {
+function cache(wrappedFn) {
```

Comment:
That's a clearer name. We can probably make it clearer still. What do you think about `vendorProcessing`?